docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-workspace-alertcenter.md

```diff
@@ -34,10 +34,11 @@ To create Google Workspace AlertCenter service account credentials:
 1. Enable **Google Workspace Alert Center API** for the Alert API. To locate this setting, you can search for "Google Workspace Alert Center API" in the search bar. Then select the **Enable** button. <br/><img src={useBaseUrl('img/send-data/google_workspace_alert_center_API_sdk.png')} alt="api-sdk" width="450"/>
 1. You will be redirected to the dashboard page. Select the **Credentials** tab in the left panel. <br/><img src={useBaseUrl('img/send-data/google_workspace_credentials.PNG')} alt="credentials" width="200"/>
 1. Click **Create Credentials**, and select **Service Account** to create service account credentials. Later you'll supply the account details and click **Done** to create a service account. <br/><img src={useBaseUrl('img/send-data/google_workspace_service_account.PNG')} alt="<service-account>" width="400"/>
-1. Navigate to the **Keys** tab on the same page. To create JSON for the service account, you must create a key. Click **Add key** and select **Create new key**. At the prompt, select **JSON** and click **Create**. <br/><img src={useBaseUrl('img/send-data/google_workspace_service_account_key.PNG')} alt="service-account-key" width="<insert-pixel-number>"/>
-1. JSON for the service account is automatically downloaded. To see what the JSON looks like, and how the JSON fields map to the fields you'll configure, see [Service account JSON](#example-of-service-account-json) below.
-1. Add domain-wide delegation to your service account using the generated client ID in the step 5 above.
-1. From the Google admin console, add your OAuth scope to the service account using the instructions [here](https://developers.google.com/admin-sdk/directory/v1/guides/delegation#delegate_domain-wide_authority_to_your_service_account) and select it in the input form. The OAuth scope for alert API is:
+1. To create JSON for the service account, you must create a key. Click the service account email to navigate to the Keys tab.<br/> <img src={useBaseUrl('img/send-data/google_workspace_service_account_create_key.png')} alt="service-account-create_key.png" width="800"/>
+1. Click **Add key** and select **Create new key**. At the prompt, select **JSON** and click **Create** to create a key. <br/><img src={useBaseUrl('img/send-data/google_workspace_service_account_key.png')} alt="service-account-key" width="600"/>
+1. JSON for the service account is automatically downloaded. To see what the JSON looks like, and how the JSON fields map to the fields you'll configure. Find the [Service account JSON](#example-of-service-account-json) example below.
```

Comment:
```suggestion
1. JSON for the service account is automatically downloaded. To see what the JSON looks like, and how the JSON fields map to the fields you'll configure, see the [Service account JSON](#example-of-service-account-json) example below.
```

Reply:
Thanks for your suggestions. You are correct because usually we use “see” when redirecting to a section or to a page. Here in this case, the example is not right after the step. So, if we use “see” here, then user will look for the example into the next step, but he won’t find it. 
“Find” will be more appropriate here, so the user will be aware that example is somewhere below in the document.